integration-cli/requirements_test.go

```diff
@@ -167,6 +167,11 @@ func DockerCLIVersion(t testing.TB) string {
 	return version[2]
 }
 
+func CgroupsV2() bool {
```

Comment:
nit: https://pkg.go.dev/github.com/containerd/cgroups/v3#Mode could be used

Reply:
Updated